.gitmodules

```diff
@@ -70,9 +70,6 @@
 [submodule "semgrep/lang"]
 	path = semgrep/semgrep/lang
 	url = https://github.com/returntocorp/semgrep-langs
-[submodule "semgrep-core/src/core/ast/lang"]
```

Comment:
you deleted to create a symlink towards ../../interface/lang which does not exist. It's not gonna work.

Reply:
oops